supply/lib/supply/uploader.rb

```diff
@@ -34,9 +34,9 @@ def perform_upload
 
     def promote_track
       version_codes = client.track_version_codes(Supply.config[:track])
-      client.update_track(Supply.config[:track], 1.0, nil)
+      client.update_track(Supply.config[:track], 0.1, nil)
```

Comment:
A comment is definitely important, probably more so than the actual value.

Please add a great comment and I'll get this merged. 👍